nautobot/extras/models/tags.py

```diff
@@ -47,6 +71,19 @@ def to_csv(self):
         return (self.name, self.slug, self.color, self.description)
 
 
+class TaggableManagerField(TaggableManager):
+    """
+    Helper class for overriding TaggableManager formfield method
+    """
+
+    def formfield(self, form_class=DynamicModelMultipleChoiceField, **kwargs):
+        kwargs["queryset"] = Tag.objects.all()
+        kwargs["required"] = False
+        kwargs["query_params"] = {"content_types": self.model._meta.label_lower}
```

Comment:
This still feels off to me - perhaps instead?
```suggestion
        kwargs.setdefault("queryset", Tag.objects.all())
        kwargs.setdefault("required", False)
        kwargs.setdefault("query_params", {"content_types": self.model._meta.label_lower})
```